warehouse/query-core/src/test/java/datawave/query/jexl/visitors/HasTopLevelNegationVisitorTest.java

```diff
@@ -9,85 +9,85 @@
 public class HasTopLevelNegationVisitorTest {
     
     @Test
-    public void test() throws Exception {
+    public void testNoNegations() throws Exception {
         ASTJexlScript script = JexlASTHelper.parseJexlQuery("FOO == 'bar'");
         Assert.assertFalse(HasTopLevelNegationVisitor.hasTopLevelNegation(script));
```

Comment:
Might as well switch these to static imports while we're here

Reply:
Went ahead and did that, thanks for reviewing.